### ISSUE TITLE
 Pretrained models for GE2E

### ISSUE THREAD

**Issue by Edresson** [clovaai/voxceleb_trainer#20]
HI, Thank you for sharing the code and pre-trained models  :)

I'm working with multi-speaker speech synthesis and I would like to know if it is possible for you to make the ResNetSE34L model trained with GE2E available, using the same parameters as the model provided trained with Angular Prototypical (EER 2.2322)

Embedding of GE2E-trained models are used to represent a speaker on multi-speaker synthesis systems, so I want to compare Angular Prototypical loss performance with GE2E in multispeaker speech synthesis :).

**Comment by joonson** [comment]
Which value of `M` would you like?

**Comment by Edresson** [comment]
Thanks for the answer. M=3

**Comment by joonson** [comment]
Try this one!

[V7902B.model.zip](https://github.com/clovaai/voxceleb_trainer/files/4625497/V7902B.model.zip)

**Comment by Edresson** [comment]
Thank you :)
